README.md

```diff
@@ -68,6 +68,8 @@ board.on('ready', function () {
 });
 ```
 
+Note, calling `analogRead`, `digitalRead` or `i2cRead` initiates a new **continuous** data reading process for a pin not a single read. Further details can be seen at https://github.com/rwaldron/io-plugins#analogreadpin-handler.
```

Comment:
"Note," isn't needed and should be removed.
The strong emphasis on **continuous** isn't needed and should be removed.
The link https://github.com/rwaldron/io-plugins#analogreadpin-handler only applies to `analogRead`, not to `digitalRead` or `i2cRead`. It should be changed to [IO Plugins](https://github.com/rwaldron/io-plugins).